examples/react-native/Intro.js

```diff
@@ -5,35 +5,35 @@
  */
  'use strict';
 
-import React, {Component} from 'react';
-import {
+ import React, {Component} from 'react';
```

Comment:
that indentation is actually wrong. Remove the extra space before `use strict` and it'll get back to normal :)